tutorials/llm/2_prompt_usage.py

```diff
@@ -0,0 +1,216 @@
+# %% [markdown]
+"""
+# LLM: 2. Prompt Usage
+
+Prompting is an essential step in using LLMs and Chatsky provides you with a
+simple way of using multiple prompts throughout your script.
+
+Using Chatsky you can specify a certain prompt
+for each flow or node to alter the model's behavior.
+"""
+
+# %pip install chatsky[llm] langchain-openai
+# %%
+
+
+import re
+
+
+from chatsky import (
+    TRANSITIONS,
+    RESPONSE,
+    GLOBAL,
+    LOCAL,
+    MISC,
+    Pipeline,
+    Transition as Tr,
+    conditions as cnd,
+    destinations as dst,
+    BaseResponse,
+    Context,
+)
+from langchain_openai import ChatOpenAI
+
+from chatsky.core.message import Message
+from chatsky.utils.testing import is_interactive_mode
+from chatsky.llm import LLM_API
+from chatsky.responses.llm import LLMResponse
+from chatsky.llm.prompt import Prompt, PositionConfig
+import os
+
+openai_api_key = os.getenv("OPENAI_API_KEY")
+
+# %% [markdown]
+"""
+Another feature is the ability to specify prompts position
+in the history that will be passed to an LLM.
+You can specify the position of the system prompt, message history
+and misc prompts, prompt specified in response
+and last message by modifying `PositionConfig`.
+
+The default positions are as follows:
+    system_prompt
+    message history
+    misc_prompt
+    call_prompt
+    last_request
+
+`LLM_API` will use these positions to order the prompts
+if not specified otherwise.
+
+Let's create a simple script to demonstrate this. Note, that prompts should go
+the `MISC` field of the node.
+Also you can alter the regular expression that is
+used to parse prompt fields in the `MISC` dictionary. By default it is "prompt"
+and can be changed by setting `prompt_misc_filter` in `LLMResponse`.
+"""
+
+# %%
+# this `system_prompt` will be always on the top of the history
+# during models response if not specified otherwise in PositionConfig
+
+# In this config `message history` will be
+# always on the second place of the history
+# misc_prompt is the default position for misc prompts
+# Misc prompts may override it and be ordered in a different way
```

Comment:
This needs clarification. And needs to be clearly different from the previous sentence (e.g. separated with a period).